### PR TITLE
Lesson-1-9 Submission

### PR DESCRIPTION
**Pull request by Tugeeta**

### CODE REVIEW

**Comment by Tugeeta** [comment]
Package. JSON and Package.lock JSON also committed along with the Lesson-1-9

File: src/App.js [hunk 1]
```diff
@@ -1,79 +1,74 @@
...
-
+  );
   const [isLoading, setIsLoading] = useState(true);
-  
-    async function fetchData (
-   
-        
-        useEffect = () => {
+
+        useEffect = (() => {
```
Comment (XeniyaDob) [inline src/App.js:12]: Please remove the extra `=` sign in the useEffect

File: src/App.js [hunk 2]
```diff
@@ -1,79 +1,74 @@
...
+            const todos = data.records.map((records) => ({
               Id: records.Id,
               Title: records.fields.Title
               
-           }
-             return (todos)      
-            
+           }));
+               
+             console.log("Todos:", todos);
```
Comment (XeniyaDob) [inline src/App.js:44]: Take a look at how you receive the data object
Screen Shot 2023-06-12 at 10 25 42 AM

You don't need to capitalize the id and title properties of the todos obj. Use them exactly as you receive them in the data obj.

File: src/App.js [hunk 3]
```diff
@@ -1,79 +1,74 @@
...
-               const data = await response.json();
-               console.log("API Data:", data);
-             };
-
+          const url = `https://api.airtable.com/v0/${process.env.REACT_APP_AIRTABLE_BASE_ID}/${process.env.REACT_APP_TABLE_NAME}`;
+             
+           const response = await fetch (url, options);
+              
              if (!response.ok) {
               const message = ('Error occurred: ${response.status}');
```
Comment (XeniyaDob) [inline src/App.js:31]: It looks like you have single quotes here instead of template literals.

**Review by XeniyaDob — CHANGES_REQUESTED** [review]
Great, you are almost there! 

I added a couple of comments to your Pull request files and also

 - Please `import React` in the `AddTodoForm.js` file.
 - You can delete the hardcoded `todoList` array in the `TodoList.js` file.

**Pull request merged**